### PR TITLE
feat: `init` can be called at any time [EXT-3817]

### PR DESCRIPTION
**Pull request by andipaetzold**
When `init` is called and no `connect` message has been received at that point in time, an `init` message is sent to the parent window.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "@types/cross-spawn": "6.0.2",
     "@types/fs-extra": "9.0.13",
     "@types/jsdom": "16.2.14",
+    "@types/mocha": "9.1.1",
```
Comment (loweisz) [inline package.json:45]: why is that needed? 🤔

Comment (andipaetzold) [reply package.json:45]: Because we currently don't have any types and the whole editor is red.
This is not directly related to the ticket, but made my life easier.

File: lib/initialize.ts [hunk 2]
```diff
@@ -36,34 +36,25 @@ export default function createInitializer(
...
 
         let customApi
         if (typeof makeCustomApi === 'function') {
           customApi = makeCustomApi(channel, params)
         }
 
-        // Handle pending incoming messages.
-        // APIs are created before so handlers are already
-        // registered on the channel.
-        messageQueue.forEach((m) => {
```
Comment (shikaan) [inline lib/initialize.ts:60]: This might be the place for the tracking, for instance

**Review by shikaan — APPROVED** [review]
The change looks good to me (thanks for the cleaning up too). I was just wondering if it's worth tracking this potentially breaking change

**Comment by andipaetzold** [comment]
This PR has been released as [4.10.0-alpha.1](https://www.npmjs.com/package/@contentful/app-sdk/v/4.10.0-alpha.1).
